src/components/screens/HomeScreen.vue

```diff
@@ -1,7 +1,7 @@
 <template>
```

Comment:
The only reason I left it was because I like the way it visually adds another column. Not sure what would go in its place, but agreed it's redundant given what information is on the stats view.